### PR TITLE
Demo defects 

### PR DESCRIPTION
**Pull request by akansha116**
QA reopen fixed

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less [hunk 1]
```diff
@@ -4,7 +4,7 @@
     h3{
         font-weight: normal;
         margin-bottom: 10px;
-         margin-top: 100px;
+          margin-top: 100px;
```
Comment (nitheshkumar-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less:7]: what is the change here

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less [hunk 2]
```diff
@@ -35,4 +35,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```
Comment (nitheshkumar-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less:38]: why we are adding and delete the braces here

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/clientlibs/less/footer.less [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
   display: block;
+    margin-top:30px;
   border-top: 1px solid #e5e5e5;
   background: #272727;
   color: #8b8b8b;
+    .cmp-image .cmp-image__image {
+    width: 100%;
+    margin-top: 45px;
+
+}
```
Comment (nitheshkumar-hcl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/clientlibs/less/footer.less]: remove the spaces

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less (hunk 1 above)
Comment (akansha116) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less:7]: it was previously added but later on some changes required so i  deleted it but asked to retain these changes so again added

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/clientlibs/less/footer.less (hunk 3 above)
Comment (akansha116) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/structure/footer/clientlibs/less/footer.less]: removed

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less (hunk 2 above)
Comment (akansha116) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/newsletter/clientlibs/less/newsletter.less:38]: only added margin-top.

**Review by nitheshkumar-hcl — APPROVED** [review]

**Pull request merged**